rspec-mode.el

```diff
@@ -146,6 +146,11 @@ Not used when running specs using Zeus or Spring."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-zeus-sock-file (concat (rspec-project-root) ".zeus.sock")
```

Comment:
> it seemed when I tested it that this was evaluated when I entered rspec-mode, and at that time spec-project-root was valid

That's also possible, if `rspec-mode` was only autoloaded when you opened the fist Ruby file. But this will break anyway as soon as you switch to a different project in the same Emacs session.